common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -120,6 +126,23 @@ def _cohort_count(self, label):
         """
         return int(label.split(' (')[1].split(')')[0])
 
+    def wait_for_cohort_management_section(self):
+        """
+        Wait for Cohort Management Section to appear.
+        """
+        self.wait_for_element_visibility(self._bounded_selector('#cohort-select'), "Select Cohort is Visible")
+
+    def save_cohort_settings(self):
+        """
+        Click on Save button shown after click on Settings tab or when we add a new cohort.
+
+        NOTE: In case of updating the existing cohort settings, Page will Reload when we click on Save button.
```

Comment:
Nonetheless, you should avoid reloading the page unless it is absolutely necessary. What was the issue you were seeing when you did not reload the page? With Backbone, it should be possible for views to update their data (and @andy-armstrong thought the views were already refreshing on Save).